docs/using.rst

```diff
@@ -5,42 +5,8 @@ User Guide
 .. contents:: Table of Contents
    :local:
 
-.. _installation:
-
-System Requirements
-===================
-
-The Let's Encrypt Client presently only runs on Unix-ish OSes that include
-Python 2.6 or 2.7; Python 3.x support will hopefully be added in the future. The
-client requires root access in order to write to ``/etc/letsencrypt``,
-``/var/log/letsencrypt``, ``/var/lib/letsencrypt``; to bind to ports 80 and 443
-(if you use the ``standalone`` plugin) and to read and modify webserver
-configurations (if you use the ``apache`` or ``nginx`` plugins).  If none of
-these apply to you, it is theoretically possible to run without root privileges,
-but for most users who want to avoid running an ACME client as root, either
-`letsencrypt-nosudo <https://github.com/diafygi/letsencrypt-nosudo>`_ or
-`simp_le <https://github.com/kuba/simp_le>`_ are more appropriate choices.
-
-The Apache plugin currently requires OS with augeas version 1.0; currently `it
-supports
-<https://github.com/certbot/certbot/blob/master/certbot-apache/certbot_apache/constants.py>`_
-modern OSes based on Debian, Fedora, SUSE, Gentoo and Darwin.
-
-
-Getting Certbot
-===============
-Certbot is packaged for many common operating systems and web servers. Check whether
-``certbot`` (or ``letsencrypt``) is packaged for your web server's OS by visiting
-certbot.eff.org_, where you will also find the correct installation instructions for
-your system.
-
-.. Note:: Unless you have very specific requirements, we kindly suggest that you use the Certbot packages provided by your package manager (see certbot.eff.org_). If such packages are not available, we recommend using ``certbot-auto``, which automates the process of installing Certbot on your system.
-.. _certbot.eff.org: https://certbot.eff.org
-
-.. _certbot-auto: https://certbot.eff.org/docs/using.html#certbot-auto
-
-Commands
-========
+The Certbot Client
```

Comment:
I'm not a big fan of this title. While Certbot is client software, "Certbot client" seems unnecessary as there is no other Certbot software such as a Certbot server. I think the wording in the beginning of the paragraph below is left over from when it was "The Let's Encrypt client" which made more sense as there is a Let's Encrypt server.

I think we should change the title to something like "Certbot", "Certbot commands", "Running Certbot", or "How to Run Certbot". I think simply "Certbot" is my personal favorite.